torch/onnx/utils.py

```diff
@@ -583,7 +583,12 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
         from torch.onnx.symbolic_helper import _export_onnx_opset_version as opset_version
         import torch.onnx.symbolic_registry as sym_registry
 
-        sym_registry.register_version('', opset_version)
+        if operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLBACK:
+            import torch.onnx.symbolic_caffe2 as sym_caffe2
+            sym_caffe2.register_quantized_ops('caffe2', opset_version)
+        else:
+            sym_registry.register_version('', opset_version)
```

Comment:
I think we should execute sym_registry.register_version('', opset_version) unconditionally.